.claude/skills/pm/SKILL.md

```
@@ -103,6 +103,15 @@ gh issue create --title "[FIX] [description]" \
    - Based on current phase and P0/P1 issues
    - Considers dependencies between tasks
    - Suggests parallel work opportunities
+   - **Always ends each task with a model/agent routing recommendation:**
+     ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)